packages/go_router/lib/src/information_provider.dart

```diff
@@ -34,6 +34,9 @@ enum NavigatingType {
   /// Restore the current match list with
   /// [RouteInformationState.baseRouteMatchList].
   restore,
+
+  /// Pop Current route.
+  pop,
```

Comment:
based on the code, this is a message to information parser that something is updating, and reparse the routeinformation should be stall. If so, we should probably using name like `stalling`, or `updating`, `skipRefresh`. in case this is used for other purpose in the future.